addons/web/static/src/legacy/utils.js

```diff
@@ -156,8 +156,9 @@ export function mapLegacyEnvToWowlEnv(legacyEnv, wowlEnv) {
             // Add user context in kwargs if there are kwargs
             if (params && params.kwargs) {
                 params.kwargs.context = Object.assign(
-                    params.kwargs.context || {},
-                    legacyEnv.session.user_context
+                    {},
+                    legacyEnv.session.user_context,
+                    params.kwargs.context,
```

Comment:
Commit [1] you mentioned is there in 15.2, there is no bug because of it in that version?

Reply:
I have not found a bug because of it in that version, but it most likely exists. I will keep the tour in master as it concerns the website-in-backend, and I will move the fix to 15.0.